tailwind.config.js

```
@@ -1,104 +1,30 @@
-          "0%, 100%": { opacity: "1" },
-          "50%": { opacity: "0.5" }
-        }
-      },
-      animation: {
-        fadeInUp: "fadeInUp 0.6s ease-out forwards",
-        "dashed-outline-enter": "dashed-outline-enter 0.8s ease-out 0.3s forwards",
-        "theme-link-enter": "theme-link-enter 0.6s ease-out forwards",
-        "underline-grow": "underline-grow 150ms ease forwards",
-        gradient: "gradient 3s ease infinite",
-        pulse: "pulse 2s cubic-bezier(0.4, 0, 0.6, 1) infinite"
+        primary: '#000000',
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)